docs/source/aws/examples/asg_not_used.rst

```diff
@@ -0,0 +1,84 @@
+AMI - ASG Garbage Collector
+====================================
+
```

Comment:
can you add the description to the pr here, ie use a narrative paragraph to explain what the policy set is doing.